tests/lib/test_io.py

```diff
@@ -123,6 +123,15 @@ def test_get_temporary_directory_path(self, get_working_path_mock):
         self.assertTrue(tmp_directory_name.startswith("test-prefix-"))
         self.assertEqual(generated_path, "/tmp/nestor/work/test-prefix-")
 
+    @patch("nestor_api.lib.io.get_working_path", autospec=True)
+    def test_get_temporary_directory_path_default(self, get_working_path_mock):
+        get_working_path_mock.return_value = "/tmp/nestor/work/1594903736050223-454134888"
+        generated_path = io.get_temporary_directory_path()
+
+        tmp_directory_name = get_working_path_mock.call_args[0][0]
+        self.assertRegex(tmp_directory_name, "^[0-9]{14,16}-[0-9]{9}$")
+        self.assertEqual(generated_path, "/tmp/nestor/work/1594903736050223-454134888")
+
```

Comment:
New lib `coverage` knows when a `if` statement is never false and won't say the if is cover when not a true + a false value, which is good 👍